packages/kit/src/cli.js

```diff
@@ -101,12 +101,12 @@ prog
 				process.stderr.write(data);
 			});
 
-			if (!watcher.vite || !watcher.vite.httpServer) {
```

Comment:
I personally think `vite` was a much clearer name. If you know it's Vite then you have a better understanding of what it is whereas `server` is much more generic. And `server.httpServer` is a little weird

Reply:
reverted